### PR TITLE
Added check for custom-url placeholder

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR adds validation for custom-url.

#### Why?

A custom-url has to contain a placeholder `*`.

### CODE REVIEW

File: src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php [hunk 1]
```diff
@@ -119,14 +120,29 @@ public function build()
                 $this->webspaces[] = $webspace;
 
                 $this->buildPortals($webspace);
-            } catch (\InvalidArgumentException $iae) {
+            } catch (\InvalidArgumentException $ex) {
```
Comment (danrot) [inline src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php]: We switched to using `$e` not `$ex`.

File: src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php [hunk 2]
```diff
@@ -119,14 +120,29 @@ public function build()
...
-                    'Error in file "' . $file->getRealPath() . '" (' . $iae->getMessage(
-                    ) . '). The file has been skipped'
+                    sprintf(
+                        'Error: "%s" in "%s". The file has been skipped.',
+                        $ex->getMessage(),
+                        $file->getRealPath()
+                    )
                 );
-            } catch (InvalidUrlDefinitionException $iude) {
+            } catch (InvalidUrlDefinitionException $ex) {
```
Comment (danrot) [inline src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php]: Same here

File: src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php [hunk 3]
```diff
@@ -119,14 +120,29 @@ public function build()
...
                 $this->logger->warning(
-                    'Error: "' . $iude->getMessage() . '" in "' . $file->getRealPath() . '". File was skipped'
+                    sprintf(
+                        'Error: "%s" in "%s". The file has been skipped.',
+                        $ex->getMessage(),
+                        $file->getRealPath()
+                    )
+                );
+            } catch (InvalidCustomUrlException $ex) {
```
Comment (danrot) [inline src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php]: Same here

**Comment by wachterjohannes** [comment]
@danrot fixed

**Pull request merged**